src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/DomainEventSubscriber.kt

```diff
@@ -15,8 +15,6 @@ data class AdditionalInformation(
   val nomsNumber: String
 )
 
-data class Event(val Message: String)
```

Comment:
moved to the `OffenderEventSubscriber` as this class will eventually disappear anyway